force-app/main/default/classes/AFFL_MultiRecordType_TEST.cls

```diff
@@ -976,4 +983,659 @@ public with sharing class AFFL_MultiRecordType_TEST {
         List<Affiliation__c> affiliationsInserted = [SELECT Id FROM Affiliation__c];
         System.assertEquals(5, affiliationsInserted.size());
     }
+
+    /**************************************************************************
+     * Unit TESTS
+     **************************************************************************/
+
+    /**
+     * @description helper class for initialization of tests for processAfflDeletedValid
+     */
+    private class ProcessAfflDeletedValidData {
```

Comment:
Is there a reason why we need a helper class other than reusability. Imho the complexity this class is adding overshadows the reusability.

Reply:
I was having a good bit of trouble balancing the data setup in each class with readability of the tests. I opted for this route to make the data setup defined and then the individual tests more transparent about exactly what they are testing.

I can flatten it out, but the complexity of these methods has required the number of unit tests. I feel like it might become even less readable.

Reply:
Do you feel like this is a case where we might benefit from separating the test classes up even further? The level of complexity has to do with the functionality. If I were to work this completely, I would refactor it into tiny parts that were much more testable. Let me know how far you feel we should stretch on this one.